planet/api/__init__.py

```diff
@@ -12,9 +12,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from .http import Session
 from .orders import OrdersClient
 from .__version__ import __version__  # NOQA
```

Comment:
This looks like it clobbers the module and makes `planet.api.__version__` a `str`, which can be confusing and/or explosive depending upon which order things are imported. For example:
```
>>> from planet.api.__version__ import __version__ as v
>>> type(v)
<class 'str'>
>>> v
'2.0.0'
>>> import planet.api.__version__ as v2
>>> v2.__version__
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'str' object has no attribute '__version__'
>>> 
```